docs/reference/setup/restart-cluster.asciidoc

```diff
@@ -8,6 +8,9 @@ nodes in the cluster while in the case of
 <<restart-cluster-rolling,rolling restart>>, you shut down only one node at a
 time, so the service remains uninterrupted.
 
+[WARNING]
+Nodes exceed low watermark will be slow to restart. You may want to reduce disk
+usage below low watermark before proceeding to restart nodes.
```

Comment:
```suggestion
[WARNING]
====
Nodes exceeding the low watermark threshold will be slow to restart. Reduce the disk
usage below the <<cluster-routing-watermark-low,low watermark>> before to restarting nodes.
====
```

Reply:
@Leaf-Lin, this wording is stronger than your original text, which I think coincides with a `WARNING` admonition. Let me know if this is a change that users should definitely make, or if we want to say that "You may want to" make this change.